pkg/apis/apis/v1alpha1/types_apiexportendpointslice.go

```diff
@@ -85,6 +92,20 @@ type APIExportEndpoint struct {
 	URL string `json:"url"`
 }
 
+func (in *APIExportEndpointSlice) GetConditions() conditionsv1alpha1.Conditions {
+	return in.Status.Conditions
+}
+
+func (in *APIExportEndpointSlice) SetConditions(conditions conditionsv1alpha1.Conditions) {
+	in.Status.Conditions = conditions
+}
+
+// These are valid conditions of APIExportEndpointSlice in addition to
+// APIExportValid and related reasons defined with the APIBinding type.
+const (
+	APIExportEndpointSliceURLsReady conditionsv1alpha1.ConditionType = "EndpointURLsReady"
+)
```

Comment:
am not convinced of adding conditions to the slice object. It's a purely data output objects, similarly to Endpoints(Slices) in kube.

What should this condition even mean? IMO there is such a think as ready for it, in the sense that ready = in-sync.

Reply:
ic, this is for `APIExport %s|%s not found` because we have an export reference in the slices. So nvmd.